### PR TITLE
additional env vars

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: docs/reference/environment-variables.md [hunk 1]
```diff
@@ -83,12 +83,21 @@ Enabling overwrites for credentials allows you to set default values for credent
...
+| :------- | :--- | :---------- |
 | `N8N_USER_MANAGEMENT_DISABLED` | Boolean | Defaults to user management enabled (false). Set to `true` to disable the [user management](user-management.md) feature. Note that n8n ignores this environment variable if you have already set up an owner account. |
-| `N8N_EMAIL_MODE` | string | smtp |
-| `N8N_SMTP_HOST` | string | _your_server_name_ |
-| `N8N_SMTP_USER` | string | _your_smtp_username_ |
-| `N8N_SMTP_PASS` | string | _your_smtp_password_ |
-| `N8N_SMTP_SENDER` | string | _your_smtp_sender_name_ |
+| `N8N_EMAIL_MODE` | string | SMTP | Required |
+| `N8N_SMTP_HOST` | string | _your_SMTP_server_name_ | Required |
+| `N8N_SMTP_PORT` | number | _your_SMTP_server_port_ | Optional |
```
Comment (Joffcom) [inline docs/reference/environment-variables.md:94]: We should add the default port to this (465)

File: docs/reference/user-management.md [hunk 2]
```diff
@@ -54,13 +54,17 @@ You need an SMTP server for user management to send invites and password resets.
...
-| `N8N_EMAIL_MODE` | string | smtp |
-| `N8N_SMTP_HOST` | string | _your_server_name_ |
-| `N8N_SMTP_USER` | string | _your_smtp_username_ |
-| `N8N_SMTP_PASS` | string | _your_smtp_password_ |
-| `N8N_SMTP_SENDER` | string | _your_smtp_sender_name_ |
+| Variable | Type | Description | Required? |
+| -------- | ---- | ----------- | --------- |
+| `N8N_EMAIL_MODE` | string | SMTP | Required |
+| `N8N_SMTP_HOST` | string | _your_SMTP_server_name_ | Required |
+| `N8N_SMTP_PORT` | number | _your_SMTP_server_port_ | Optional |
```
Comment (Joffcom) [inline docs/reference/user-management.md:61]: We should add the default port to this (465)

**Review by Joffcom — CHANGES_REQUESTED** [review]
Only 2 changes to make to add the default SMTP port.

File: docs/reference/environment-variables.md (hunk 1 above)
Comment (StarfallProjects) [reply docs/reference/environment-variables.md]: added

File: docs/reference/user-management.md (hunk 2 above)
Comment (StarfallProjects) [reply docs/reference/user-management.md]: added

**Pull request merged**